components/webFeatures.ts

```diff
@@ -50,7 +58,7 @@ webFeaturesRouter.get(
             !req.query.gv ||
             !["h1", "h2", "h3"].includes(req.query.gv ?? null)
         ) {
-            res.json({ error: "bad gv" })
+            res.json([])
```

Comment:
Why return an empty list here when using that error helper everywhere else?

Reply:
Partly because it is supposed to return a list here, but also because `gv` can be `h0` here. Another workaround is add `h0` as a potential value for a `GameVersion`, return an empty list if the `gv` is `h0`, then use that error helper if the `gv` is any other value.